java/src/com/ibm/streamsx/rest/package-info.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * # Licensed Materials - Property of IBM # Copyright IBM Corp. 2017
+ */
+
+/**
+ * Java wrappers for the REST API of IBM Streams.
+ * 
+ * <p>
```

Comment:
anywhere you start a paragraph with `<p>` you need to terminate it with `</p>`, otherwise accessibility checking tools raise errors.